cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/KafkaRollerTest.java

```diff
@@ -206,11 +207,21 @@ public void controllerNotReadyAfterRolling(VertxTestContext testContext) throws
                 singletonList(2));
     }
 
+    public List<String> addPodNames(int replicas) {
+        ArrayList<String> podNames = new ArrayList<>(replicas);
+
+        for (int podId = 0; podId < replicas; podId++) {
+            podNames.add(KafkaCluster.kafkaPodName(clusterName(), podId));
+        }
+
+        return podNames;
+    }
```

Comment:
I think (one of the) proper tests here would be to try it with disconnected list of names. E.g. `my-cluster-kafka-0`, `my-cluster-kafka-1000`, `my-cluster-kafka-controller-2000` for example and check they were all rolled.

Reply:
> I think (one of the) proper tests here would be to try it with disconnected list of names. E.g. my-cluster-kafka-0, my-cluster-kafka-1000, my-cluster-kafka-controller-2000 for example and check they were all rolled.

Might be still useful to try this - at least in some tests ^^^

Reply:
Should I create a separate test which do this?

Reply:
I think as a separate test it would be fine, used in the existing tests is fine as well.

Reply:
I'd prefer changing the existing test.